packages/plugins/chugsplash/Storage.config.ts

```diff
@@ -10,11 +10,15 @@ const config: UserChugSplashConfig = {
   contracts: {
     MyStorage: {
       contract: 'Storage',
-      variables,
+      variables: {
+        ...variables,
+        myStorage: '0x1111111111111111111111111111111111111111',
+      },
     },
     MySimpleStorage: {
-      contract: 'SimpleStorage',
+      contract: 'Storage',
       variables: {
+        ...variables,
```

Comment:
Is there a reason you changed this from `SimpleStorage`? If you don't want to use `SimpleStorage.t.sol`, could you remove it?

Reply:
I actually changed this because `SimpleStorage.t.sol` has been removed already. I'm not sure why though, I was going to ask you about that actually haha.

Reply:
Whoops, my bad! Added that back